.github/CODEOWNERS

```diff
@@ -108,8 +108,9 @@
 # PRLabel: %Search
 /sdk/search/ @xirzec @sarangan12
 
-/sdk/applicationinsights/applicationinsights-query/ @divyajay @alongafni
-/sdk/operationalinsights/loganalytics/ @divyajay @alongafni
+# PRLabel: %Monitor - ApplicationInsights
+# ServiceLabel: %Monitor - ApplicationInsights %Service Attention
+/sdk/applicationinsights/Microsoft.Azure.App*/  @omziv @anatse @raronen @ischrei @danhadari @azmonapplicationinsights
```

Comment:
There is no folder called `Microsoft.Azure.App` in this repo. See https://github.com/Azure/azure-sdk-for-js/tree/main/sdk/applicationinsights